index.html

```diff
@@ -65,7 +65,7 @@ <h1 class="nam">Tonic</h1>
                 <li class="skill_tab">Html</li>
                 <li class="skill_tab">Javascript</li>
             </ul>
-            <div class="bttn"><a href="#" class="bcolor">See Project</a></div>
+            <div class="bttn">See Project</div>
```

Comment:
- [ ] Well done so far with the required changes. The design looks good now. However, you're still missing `source code` button beside `see project` button, please refer to the previous reviewer's comment and check the highlighted pic from the original design. 🙏